makefile

```diff
@@ -863,7 +863,7 @@ ifeq ($(platform),windows)
 	exe-suffix = .exe
 	rpath =
 
-	lflags = -L$(lib) $(common-lflags) -lws2_32 -liphlpapi -mconsole
+	lflags = -L$(lib) -static-libgcc -static-libstdc++ -Wl,-Bstatic -lstdc++ -lpthread -Wl,-Bdynamic $(common-lflags) -lws2_32 -liphlpapi -mconsole
```

Comment:
What requires libstdc++ in the build?  Once upon a time, at least, we didn't depend on that.  I'd like to keep it that way, if possible.